source/develop/release-management/features/virt/parallel-migration-connections.md

```diff
@@ -0,0 +1,88 @@
+
+- There is very little documentation about parallel migration connections and the information above was obtained directly from platform developers.
+
+## User experience
+
+A new migration option to enable parallel migration connections is introduced in cluster and VM edit dialogs.  The cluster option specifies what to use in the given cluster.  VMs can override the cluster value individually.  The new option offers the following choices:
+
+- **Disabled** (the default in clusters): Parallel migration connections are not used.
+
+- **Auto**: Use parallel migration connections (TODO: Even when there is insufficient network bandwidth, for the supposed benefit of a more efficient implementation?) and set their number automatically based on the available resources (network bandwidth and CPU power).
+
+- **Custom**: Use parallel migration connections and use the number of connections specified by the user.  The minimum number of connection is 2.  (TODO: Should we put any real limit on the maximum number?)
+
+- **Cluster value** (only in the VM edit dialog; it's the default there): Use parallel connections as specified in the VM's cluster settings. 
+
+We use **Disabled** as the cluster default because it's a newly introduced feature in oVirt and there is not enough experience with it.  We may consider changing it to **Auto** in next releases, especially if QEMU decides using parallel migration connections by default.
+
+If **Auto** is selected then the number of parallel connections is set to the lower number of the currently available migration network bandwidth divided by 10 Gbps and the number of VM vCPUs, but no less than 2.
+
+If parallel migration connections are enabled, the following adjustments are made when a migration is initiated:
+
+- Compression is disabled regardless of the selected migration policy.  (TODO: Check whether we can enable gzip/zstd compression.  If we can then it may be a good idea to always enable the compression.)
+
+- The maximum number of outgoing migrations is set to 1 regardless of the selected migration policy.  This may not be optimal if there is a non-parallel migration running that cannot saturate the available bandwidth but even in such a case it's generally simpler and safer not to start competing for the bandwidth.
+
+- If post-copy migration policy is selected, non-parallel migration is used regardless the settings.  (TODO: This is not the best user experience, it would be better to warn about this in the edit dialogs.  But how to handle the case when the cluster migration policy is non-post-copy, parallel connections are disabled in the cluster and parallel connections are enabled in some VM, which is valid so far, and then the cluster migration policy is changed to post-copy, which is not valid for the given VM anymore?)
```

Comment:
We can check settings for all the VMs in the cluster. It's still a question whether forcing the user to disable contingent parallel connections for all the affected VMs is the best user experience. It would most likely not be a typical use case so perhaps it can be acceptable.

Reply:
OK, done.